metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj

```diff
@@ -311,32 +308,32 @@
     (testing "find with parameters"
       (testing "latest revsions"
         (are3 [granules params]
-              (= (set (map util/expected-concept granules))
+          (is (= (set (map util/expected-concept granules))
```

Comment:
Nice catch on old tests!

Reply:
Thanks. It is pretty crazy that we bypassed most of the assertions in this test file when we changed are2 to are3.